tests/unit/dp/custom_dp/covariance_custom_test.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+
+from inst.python.dp.custom_dp.covariance_custom import custom_bounded_covariance, noisy_average
+
+
+def test_noisy_average():
+    x = np.array([167, 192, 173, 174, 172, 167, 171, 185, 163, 170])
+
+    n = len(x)
+    n_noisy = n
+
+    x_min = 150
+    x_max = 200
+
+    epsilon = 100
+
+    x_noisy_average = noisy_average(x, epsilon, x_min, x_max, n, n_noisy)
+    x_average = np.mean(x)
+    assert np.abs(x_noisy_average - x_average) / x_average < 0.01
```

Comment:
What do you think about putting 0.1 in a global variable and create a util function to compute difference between true and noisy computations? 
It would be more meaningful and could be reused in `test_custom_bounded_covariance` and later tests